File: MPI/mpicheck/tests/functions/2.0/MPI_File_set_view.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_File var_0;
    MPI_Offset var_1;
    MPI_Datatype var_2;
    MPI_Datatype var_3;
    const char *var_4;
    MPI_Info var_5;
    int ret;
    /* calls */
    ret = MPI_File_set_view(var_0, var_1, var_2, var_3, var_4, var_5);
    ret = PMPI_File_set_view(var_0, var_1, var_2, var_3, var_4, var_5);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_File_set_view.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER(KIND=MPI_OFFSET_KIND) var_1
       INTEGER var_2
       INTEGER var_3
       CHARACTER*(10) var_4
       INTEGER var_5
       INTEGER var_6
        call mpi_file_set_view(var_0, var_1, var_2, var_3, var_4, var_5, var_6)
       call pmpi_file_set_view(var_0, var_1, var_2, var_3, var_4, var_5, var_6)
        end program main
    